### DISCUSSION TITLE
Updating Foundation for Emails from 2.2.0 to 2.3.1

### DISCUSSION THREAD

**Discussion by toulouse88** [foundation/foundation-emails#1088] [Q&A]
We've been using Foundation for Emails 2.2.0 and we want to upgrade to 2.3.1. How exactly should that be done? Do we just copy in certain files (scss etc) in our existing project, or do we need to do a full uninstall/install of the Foundation package?

**Comment by joeworkman** [comment]
You can simply upgrade the email package. `npm upgrade foundation-emails`

**Reply by toulouse88** [reply]
Thanks!
